experimental/packages/opentelemetry-sdk-metrics-base/src/view/View.ts

```diff
@@ -14,53 +14,107 @@
  * limitations under the License.
  */
 
+import { PatternPredicate } from './Predicate';
+import { AttributesProcessor, FilteringAttributesProcessor } from './AttributesProcessor';
+import { InstrumentSelector } from './InstrumentSelector';
+import { MeterSelector } from './MeterSelector';
 import { Aggregation } from './Aggregation';
-import { AttributesProcessor } from './AttributesProcessor';
+import { InstrumentType } from '../InstrumentDescriptor';
 
-// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view
+export type ViewOptions = {
+  // View
+  /**
+   *  If not provided, the Instrument name will be used by default. This will be used as the name of the metrics stream.
+   */
+  name?: string,
+  /**
+   * If not provided, the Instrument description will be used by default.
+   */
+  description?: string,
+  /**
+   * If provided, the attributes that are not in the list will be ignored.
+   * If not provided, all the attribute keys will be used by default.
+   */
+  attributeKeys?: string[],
+  /**
+   * The {@link Aggregation} aggregation to be used.
+   */
+  aggregation?: Aggregation,
 
-export interface ViewStreamConfig {
+  // Instrument
+  /**
+   * The type of the Instrument(s).
```

Comment:
We need to clarify that these instrument and meter fields are selectors.

Reply:
Good point. :+1: Added some more documentation in [59e69ae](https://github.com/open-telemetry/opentelemetry-js/pull/3066/commits/59e69ae64659b985b246d27a4051b612cc7ae9c5). :slightly_smiling_face: